### PR TITLE
Updated go to 1.19.7

### PR DESCRIPTION
**Pull request by jmhbnz**
This pull request proposes to update to align with go versions used by `etcd` as  Go `1.17` is out of support, this brings `bbolt` in line with other etcd maintained projects and addresses CVEs with older versions of golang.

### CODE REVIEW

File: CHANGELOG/CHANGELOG-1.3.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Note that we start to track changes starting from v1.3.7.
 - Fix [open db file readonly mode for commands which shouldn't update the db file](https://github.com/etcd-io/bbolt/pull/365), see also [pull/292](https://github.com/etcd-io/bbolt/pull/292).
 
 ### Other
-- [Build bbolt CLI tool, test and format the source code using golang 1.17.13](https://github.com/etcd-io/bbolt/pull/297).
+- [Build bbolt CLI tool, test and format the source code using golang 1.19.7](https://github.com/etcd-io/bbolt/pull/409).
```
Comment (ahrtr) [inline CHANGELOG/CHANGELOG-1.3.md:29]: Please revert this change for now, and do it when we bump go version for [release-1.3](https://github.com/etcd-io/bbolt/tree/release-1.3)

**Comment by ahrtr** [comment]
@ptabor any immediate concern on this PR? It seems OK to me.

File: CHANGELOG/CHANGELOG-1.3.md (hunk 1 above)
Comment (jmhbnz) [reply CHANGELOG/CHANGELOG-1.3.md]: Good point, sorry getting ahead of myself! 🤦🏻‍♂️

**Review by ptabor — APPROVED** [review]

**Comment by ptabor** [comment]
For 1.4 no concerns.
For 1.3 -> I don't think we need to drive up the requirements. It's up to the user of the library to pick up the golang version.
